Observer/src/observer/Subscriber.java

```diff
@@ -0,0 +1,22 @@
+package observer;
+
+public class Subscriber {
+    private String name;
+    private Channel channel = new Channel();
```

Comment:
subscriber should be able to subscribe to multiple channels

Reply:
Done